packages/expo-updates/android/src/main/java/expo/modules/updates/EnabledUpdatesController.kt

```diff
@@ -95,6 +101,7 @@ class EnabledUpdatesController(
       startupFinishedMutex.withLock {
         if (!startupFinishedDeferred.isCompleted) {
           startupFinishedDeferred.complete(Unit)
+          UpdatesControllerRegistry.metricsController = WeakReference(this@EnabledUpdatesController)
```

Comment:
I think this diverges from the iOS implementation since the property isn't set until post-startup-procedure. Should it happen during initialization the same as iOS?